### ISSUE TITLE
Javadocs and comments

### ISSUE THREAD

**Issue by AbbyCottontail** [Mannjamin/Ducky-Mc-Duckerson#15]
Comments are currently all over the place and there's currently a lack of javadocs in code. At some point there need to be javadocs and comments generated for the codes to help future coders on ducky.

**Comment by LucileDT** [comment]
Comments generated are often not really helping, don't hesitate to right them manually.

I think you need from now, each time you create a function, to add the javadoc for it *(in my IDE, write `/**`then press `enter` automatically create the javadoc comment block and I only have to edit it to add useful information like what each parameters are, what the method do, ...)*. 
You can also try to create a bloc of documentation for function that you often use 🙂 

I installed once [Sonar](https://www.sonarqube.org), it displays nice graphs which show how much of your code have documentation, is tested... Maybe you can try it to have an overview!

**Comment by AbbyCottontail** [comment]
I agree with that and have started doing Java doc as best as I can, although I will admit part of my problem is I'm all over the place.
